www/on/npm/index.html.spt

```diff
@@ -1,9 +1,129 @@
+from gratipay.utils import icons
 [---]
-title = "Search npm"
+banner = manager = "npm"
 suppress_sidebar = True
-website.redirect('/')  # nothing here yet
+npackages, Npackages = website.db.one('''
+    select (select count(*) from teams_to_packages) as n, (select count(*) from packages) as t
+''')
+if user.participant:
+    packages_for_claiming = user.participant.get_packages_for_claiming(manager)
+    any_claimable = any([rec.claimed_by is None for rec in packages_for_claiming])
 [---]
 {% extends "templates/base.html" %}
+
+{% block banner %}
+<a class="elsewhere" href="https://www.npmjs.com/">
+    <div class="avatar">
+        <img class="avatar" src="{{ website.asset('package-default-large.png') }}" />
+        <img class="platform" src="{{ website.asset('npm-n.png') }}" />
+    </div>
+    {{ super() }}
+</a>
+{% endblock %}
+
+{% block scripts %}
+<script>
+    $(document).ready(function() {
+        Gratipay.packages.initBulk();
+    });
+</script>
+{{ super() }}
+{% endblock %}
+
 {% block content %}
-<input type="text" name="q"> <button class="selected">Search</button>
+
+<p class="description important-thing-at-the-top">
+    {{ _('Free as in money.') }}
+</p>
+
+<p class="instructions">
+    {{ _('Apply to accept payments for your npm packages:') }}
+</p>
+
+{% if user.ANON %}
+    <div class="important-button">
+        {{ sign_in_using(button_class='large') }}
+    </div>
+{% else %}
+
+    {% if not packages_for_claiming %}
+    <p class="sorry">{{ _("No packages found.") }}</p>
```

Comment:
Might be worth adding a note saying that we searched using the email addresses a, b, and c - and to update on npm if needed (just like the page for a single package)